metagraph/src/graph/alignment/dbg_aligner.cpp

```diff
@@ -63,7 +63,7 @@ ::align_batch(const std::vector<IDBGAligner::Query> &seq_batch,
         auto seeder = build_seeder(this_query, is_reverse_complement, nodes);
         auto extender = build_extender(this_query, aggregator);
 
-        size_t num_explored_nodes = 0;
+        double num_explored_nodes = 0;
```

Comment:
why double?

Reply:
For readability, this should probably stay as `size_t`. I changed it to clean up some of the `static_cast`s in the log message below.

Reply:
I changed this back and kept the static_cast